### DISCUSSION TITLE
Firmware update with ethLoader and canLoader

### DISCUSSION THREAD

**Discussion by KevinAStein** [robotology/community#91] [Q&A] [answered]
Hi,
we are just updating the firmware and were wondering if it is possible to upload the ems.hex for all boards in one go.
Same question for canLoader. Can we tick all boxes and upload (e.g. strain.hex)?
Thanks
@hu-yue

**Comment by randaz81** [comment, marked as answer]
Hi, for the EMS yes, you can tick all the boards, since the firmware is the same (ems.hex) for all of them.
For the canLoader, it's possible (in principle). However, except for the strain board (strain.hex), all the other boards have different firmware types.

**Comment by KevinAStein** [comment]
@randaz81 thanks, but in the canLoader there is also only one 2FOC.hex, skin.hex and strain.hex file, is it not possible, for example on the board 10, check all the boxes and load the skin.hex at once?

**Comment by pattacini** [comment]
@randaz81 ?

**Comment by julijenv** [comment]
yes you can load in one shot all the skin parts (with skin.hex)

**Comment by pattacini** [comment]
Thanks @julijenv.
